packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -122,6 +122,7 @@ export function Initialization(options?: {
         },
         bubbles: true,
         cancelable: true,
+        composed: true,
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Event/composed

Reply:
Won't be able to compose the atomic-text component inside another atomic component without it

Reply:
I'll add a comment